internal/bminventory/inventory.go

```diff
@@ -65,6 +65,9 @@ const (
 const DefaultUser = "kubeadmin"
 const ConsoleUrlPrefix = "https://console-openshift-console.apps"
 
+// 125 is the generic exit code for cases the error is in podman / docker and not the container we tried to run
+const containerAlreadyRunningExitCode = 125
```

Comment:
didn't we saw 137 too?

Reply:
maybe it is not really interesting to verify error code?

Reply:
125 is a generic error code https://docs.docker.com/engine/reference/run/#exit-status